source/api/csw.rst

```diff
@@ -2,3 +2,61 @@
 
 Catalog Service for the Web (CSW)
 #################################
+
+The CSW end point exposes the metadata records in your catalog in XML format using the OGC CSW protocol (version 2.0.2). 
+
+Two protocols are available:
+
+ - CSW: Provides the ability to search and publish metadata for data, services and related information.
+ - CSW-T: Provides an interface for creating, modifying and deleting catalog records via the CSW protocol.
+
+Configuration
+=============
+
+The CSW Service is configured in ``CSW`` in the ``Settings`` section of the Admin Dashboard.
+
+.. figure:: img/csw-configure.png
```

Comment:
For this kind of cases. Does it make sense to improve the help text in the app and only put a screenshot in the doc?